src/overview/index.md

```diff
@@ -1,21 +1,33 @@
 # Ecosystem Overview
 
-There are two approaches for using Rust on Espressif chips:
+There are the following approaches to use Rust on Espressif chips:
 
-1. _With_ the full standard library available (`std`)
-2. _Without_ the standard library available (`no_std`)
+- Using the `standard` library (`std`)
+- Using the `core` library (`no_std`), a.k.a. bare metal development
```

Comment:
I would suggest adding a link to the [`standard`](https://doc.rust-lang.org/std/) and [`core`](https://doc.rust-lang.org/core/) documentation and also using `std` instead of `standard`: 
```suggestion
- Using the `std` library, a.k.a Standard Library.
- Using the `core` library (`no_std`), a.k.a. bare metal development
```